copyparty/docker-compose.yml

```
@@ -0,0 +1,18 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: copyparty_web_1
+      APP_PORT: 3923
+
+  web:
+    image: ghcr.io/9001/copyparty-ac:1.19.17@sha256:4fed30b24b217378314e10ebfbc67ad6c02be89734e8735c0b65f0dd8e524e0b
```

A new version is available, we can update to that
```suggestion
    image: ghcr.io/9001/copyparty-ac:1.19.19 @sha256:0799b3a495e4ba97304d8eeb052aafec744d871bdcbb0ad996a04c573d3ba696
```